tests/duck.spec.js

```diff
@@ -0,0 +1,41 @@
+const { test, expect } = require('@playwright/test');
```

Comment:
After success review please merge your PR and then create new one for further tasks.